### PR TITLE
Plugin html choice

### PR DESCRIPTION
**Pull request by younesStrittmatter**
Clickable HTML elements to present choices. The example shows a use case for this plugin to present a four-arm bandit task.

### CODE REVIEW

File: packages/plugin-html-choice/package.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@jspsych-contrib/plugin-html-choice",
+  "private": "true",
```
Comment (jodeleeuw) [inline packages/plugin-html-choice/package.json:3]: this flag will need to be removed

File: .changeset/tasty-pandas-cry.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@jspsych-contrib/plugin-html-choice": patch
```
Comment (jodeleeuw) [inline .changeset/tasty-pandas-cry.md:2]: should this be a major release, or are you intending this to be version 0.1.1 of the plugin?

**Comment by younesStrittmatter** [comment]
I implemented the requested changes

**Comment by younesStrittmatter** [comment]
I think I implemented all the changes?

**Comment by jodeleeuw** [comment]
Sorry, I lost track of this! Merging in now.

**Pull request merged**